client/components/fields/editor/base/dateTimeUIFramework.tsx

```diff
@@ -55,10 +57,17 @@ export class EditorFieldDateTimeUIFramework extends React.PureComponent<IProps>
                     error={error}
                     disabled={this.props.disabled}
                     dateFormat={appConfig.planning.dateformat}
-                    value={value == undefined ? null : new Date(value)}
+                    value={
+                        value == undefined
```

Comment:
should be `value != null` or `typeof === 'undefined'` if it's important if it's `null` or `undefined`